Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/livy/interactive/Session.java

```diff
@@ -240,6 +265,7 @@ private PostSessions preparePostSessions() {
         entity.setContentType("application/json");
 
         return getHttp()
+                .setUserAgent(getUserAgent(true))
```

Comment:
NIT: Why not put `setUserAgent` into function `getHttp` if we have to set UserAgent when calling `getHttp`?

Reply:
In case we need to decide to record the request and install ID mapping or not.